README.md

```diff
@@ -28,8 +26,43 @@ Install the migrations
 rails bulkrax:install:migrations db:migrate
```

Comment:
take this install migrations step out as its not actually needed. Just have them run db:migrate.

Reply:
removed migration step